example/src/App.tsx

```diff
@@ -88,21 +85,11 @@ export default function App() {
           deviceId,
         });
         setLocalVideoTrack(createdLocalVideoTrack);
-        localVideoTrackResolve(createdLocalVideoTrack);
       } catch (error) {
         console.log(error);
       }
     };
     asyncCreateLocalVideoTrack();
-
-    return () => {
-      const cleanupAsync = async () => {
-        const createdLocalVideoTrack = await localVideoTrackPromise;
-        createdLocalVideoTrack.destroy();
-        setLocalVideoTrack(undefined);
-      };
-      cleanupAsync();
```

Comment:
these cleanups were causing issues in the OG web impl too since they'd try to run after the component was unmounted.